### DISCUSSION TITLE
frigate-hass-integration backward compatibility

### DISCUSSION THREAD

**Discussion by kingtat** [blakeblackshear/frigate#15001] [Ask A Question] [answered]
I prefer the legacy UI for my situation and wondered if the recent versions of the integration will continue to work with it.
Can I use frigate-hass-integration >=5.4.1 with frigate 0.13.2?
Thanks

**Comment by NickM-27** [comment, marked as answer]
new features won't but in general it will work fine, though the upcoming frigate 0.15 (currently in beta) has a new Explore view that users who don't prefer the reviews system have said works better than the old UI for their use cases

**Comment by kingtat** [comment]
Thanks @NickM-27. That's good to know and in that case I am looking forward to trying the 0.15 release!
